### PR TITLE
NFSoRDMA Fixes

### PR DESCRIPTION
**Pull request by Gaurav-Gangalwar**
NFSoRDMA fixes

### CODE REVIEW

**Comment by Gaurav-Gangalwar** [comment]
@ffilz @dang please review

**Comment by dang** [comment]
You don't need to tag me, I get all notifications for NTIRPC.

**Review by dang — APPROVED** [review]

**Comment by Gaurav-Gangalwar** [comment]
Looks like we merged out of order
I added comment also https://github.com/nfs-ganesha/ntirpc/pull/299#pullrequestreview-2065894178

**Comment by Gaurav-Gangalwar** [comment]
@ffilz please help with merge, It has some important fixes to avoid crashes on disconnect.

**Comment by ffilz** [comment]
Could you rebase this please?

**Comment by Gaurav-Gangalwar** [comment]
> Could you rebase this please?

Ok, I will rebase and update the patch

File: src/svc_xprt.c [hunk 1]
```diff
@@ -152,9 +152,8 @@ svc_xprt_lookup(int fd, svc_xprt_setup_t setup)
 		return (NULL);
 
 	sk.xprt.xp_fd = fd;
-#ifdef USE_RPC_RDMA
```
Comment (rojingeorge) [inline src/svc_xprt.c:155]: Please put back this USE_RPC_RDMA, else compilation fails for non-RDMA paths

**Review by rojingeorge — APPROVED** [review]

**Pull request merged**